### PR TITLE
Issue #7 customize project template

### LINKED ISSUE #7: Customize the template project for bichitos :D

### PR DESCRIPTION
**Pull request by lpardey**
- Add content to readme.md
- Update pyproject toml basic info
- Update devcontainer.json

closes #7

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
 				"[python]": {
 					"editor.formatOnSave": true,
 					"editor.codeActionsOnSave": {
-						"source.organizeImports": true
+						"source.organizeImports": "explicit"
```
Comment (rogerclotet) [inline .devcontainer/devcontainer.json:19]: What does this do?

**Review by rogerclotet — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (lpardey) [reply .devcontainer/devcontainer.json:19]: It organizes imports explicitly on file save. The 'true' value will be deprecated in favor of 'explicit'.

Comment (lpardey) [reply .devcontainer/devcontainer.json:19]: > What does this do?
It organizes imports explicitly on file save. The 'true' value will be deprecated in favor of 'explicit'.

Comment (lpardey) [reply .devcontainer/devcontainer.json:19]: It organizes imports explicitly on file save. The 'true' value will be deprecated in favor of 'explicit' so I changed it.

**Pull request merged**